refinery/ui/source/js/data-set-import/controllers/isa-tab-import.spec.js

```diff
@@ -0,0 +1,89 @@
+(function () {
+  'use strict';
+
+  describe('IsaTabImportCtrl', function () {
+    var ctrl;
+    var importConfirmationService;
+    var isaTabImportApiService;
+
+    beforeEach(function () {
+      module('refineryApp');
+      module('refineryDataSetImport');
+      inject(function ($controller, $rootScope, _isaTabImportApi_, _importConfirmationService_) {
+        isaTabImportApiService = _isaTabImportApi_;
+        importConfirmationService = _importConfirmationService_;
+        ctrl = $controller('IsaTabImportCtrl', { $scope: $rootScope.$new() });
+      });
+    });
+
+    it('should exist', function () {
+      expect(ctrl).toBeDefined();
+    });
+
+    describe('vm.showFileUpload', function () {
+      it('should be initially false', function () {
+        expect(ctrl.showFileUpload).toEqual(false);
+      });
+    });
+
+    describe('vm.dataSetUUID', function () {
+      it('should be initially undefined', function () {
+        expect(ctrl.dataSetUUID).toEqual(undefined);
+      });
+    });
+
+    describe('vm.isMetaDataRevision', function () {
+      it('should be initially false', function () {
+        expect(ctrl.isMetaDataRevision).toEqual(false);
+      });
+    });
+
+    describe('vm.isaTabImportApi', function () {
```

Comment:
We typically don't check for dependencies in unit tests. Is this a common thing elsewhere?

Reply:
Not sure, I was following a pattern I saw in another test file.